pkg/controller/clusterautoscaler/clusterautoscaler_controller.go

```diff
@@ -35,6 +36,7 @@ const (
 
 // NewReconciler returns a new Reconciler.
 func NewReconciler(mgr manager.Manager, config Config) *Reconciler {
+	monitoringv1.AddToScheme(mgr.GetScheme())
```

Comment:
I would probably just do this where we instantiate the operator and manager: [here](https://github.com/openshift/cluster-autoscaler-operator/blob/master/pkg/operator/operator.go#L65)